subresource-loading.bs

```diff
@@ -605,18 +605,16 @@ To <dfn>find a matching web bundle registration</dfn> given [=request=]
 
    1. Let |rule| be |registration|'s [=web bundle registration/rule=].
 
-   2. If |url| doesn't meet a path restriction rule, given |registration|'s
-      [=web bundle registration/fetch entry=]'s [=web bundle fetch
-      entry/source=], then return null.
+   2. Let |allowed path| be the result of [shortening]
+      (https://url.spec.whatwg.org/#shorten-a-urls-path) |registration|'s [=web
```

Comment:
This is not linkified in the rendered HTML...

Reply:
Sorry. Fixed.